case-lib/lib.sh

```diff
@@ -397,7 +399,27 @@ is_zephyr()
 
 logger_disabled()
 {
-    [[ ${OPT_VAL['s']} -eq 0 ]]
+    local ldcFile
+    # Some firmware/OS configurations do not support logging.
+    ldcFile=$(find_ldc_file) || {
+        dlogi '.ldc dictionary file not found, SOF logs collection disabled'
+        return 0 # 0 is 'true'
```

Comment:
Not sure this comment is needed, as the script self documents on the line before.

Reply:
You're right, it's overkill. I will remove but only if someone requests some other changes because it does do not much harm either and uses very little real estate.